### ISSUE TITLE
Please change libxmljs to latest version ^0.19.0 

### ISSUE THREAD

**Issue by mikerandall** [trevordixon/excel.js#64]
Using version `^0.18.0` grabs version `0.18.8` which uses `node-pre-gyp` version `^0.9.1`. This version [has been giving users this issue which has been fixed in version 0.10](https://github.com/mapbox/node-pre-gyp/issues/367).

**Comment by XiaoJianPeng** [comment]
Please change libxmljs to latest version
`npm install excel ` 
```
node-pre-gyp http 404 https://github.com/libxmljs/libxmljs/releases/download/v0.18.8/node-v64-win32-x64.tar.gz
node-pre-gyp ERR! Tried to download(404): https://github.com/libxmljs/libxmljs/releases/download/v0.18.8/node-v64-win32-x64.tar.gz
```

**Comment by clickclickonsal** [comment]
libxmljs is no longer used in the project so I accepted PR #61 which removes this as a dependency

**Comment by clickclickonsal** [comment]
I published version 1.0.1 that includes this fix.
